torch/_dynamo/graph_deduplication.py

```diff
@@ -236,12 +239,12 @@ def _create_subgraph(
 
 def _args(
     n: torch.fx.Node,
-    node_to_additional_deps: Optional[dict[torch.fx.Node, list[torch.fx.Node]]] = None,
-) -> list[torch.fx.node.Argument]:
+    node_to_additional_deps: Optional[dict[Node, OrderedSet[Node]]] = None,
+) -> list[Any]:
```

Comment:
Why reduce to Any? Is there no baseclass that is valid?

Reply:
Yeah the arguments can be anything